pkg/rpc/server/server_test.go

```diff
@@ -2464,17 +2469,25 @@ func checkNep17Balances(t *testing.T, e *executor, acc interface{}) {
 			{
 				Asset:       rubles,
 				Amount:      "877",
+				Decimals:    2,
 				LastUpdated: 6,
+				Name:        "Rubl",
+				Symbol:      "RUB",
 			},
 			{
 				Asset:       e.chain.GoverningTokenHash(),
 				Amount:      "99998000",
 				LastUpdated: 4,
+				Name:        "NeoToken",
+				Symbol:      "NEO",
```

Comment:
Why don't we have decimals here?

Reply:
NEO is non-divisible, so `Decimals` is zero which is the default value anyway.